apps/gallery/style/gallery.css

```diff
@@ -894,13 +894,13 @@ html[dir="rtl"] .sliderlabel {
   background: url(images/slider-marker.png) no-repeat top right;
 }
 
-.sliderlabel:after {
+.sliderlabel bdi:after {
   content: attr(data-label);
   position: relative;
   text-align: center;
   top: 0.9rem;
   width: 1rem;
-  -moz-margin-start: calc(-50%  + 0.1rem);
+  margin: 0 calc(-50% + 0.1rem);
```

Comment:
This makes me nervous because it assumes that a bdi:after pseudo-element will have the same bidi behavior as the bdi element itself.

Instead, please try just adding unicode-bidi: isolate to the sliderlabel:after styles, and don't use the bdi elements at all.  That seems like cleaner markup and puts the isolation on exactly the element we want.